src/aiortc/rtcconfiguration.py

```diff
@@ -64,3 +82,6 @@ class RTCConfiguration:
 
     bundlePolicy: RTCBundlePolicy = RTCBundlePolicy.BALANCED
     "The media-bundling policy to use when gathering ICE candidates."
+    
+    socks5Proxy: Optional[RTCSocks5Proxy] = None
+    "A :class:`RTCSocks5Proxy` object to configure a SOCKS5 proxy for all UDP traffic."
```

Comment:
This doesn't exist in the "normal" `RTCConfiguration` object. It feels as though this should be configured differently, possibly through an environment variable?

@fippo ?